connect-transformation/solutions/resources/jdbc-source-connector.yaml

```diff
@@ -16,7 +16,11 @@ spec:
     table.whitelist: accountants
     mode: incrementing
     incrementing.column.name: id
-    transforms: MaskField
-    transforms.MaskField.type: org.apache.kafka.connect.transforms.MaskField$Value
-    transforms.MaskField.fields: ssn
+    # TODO: Add Single Message Transformations
+    transforms: Mask, Insert
```

Comment:
These spaces are tricky. In the book, we also have a space after "Mask", like: `Mask , Insert`. The some worked for me when testing the lab